config/github_runner_labels.yml

```diff
@@ -1,35 +1,37 @@
+- { name: ubicloud-standard-16-ubuntu-2004,     vm_size: standard-16, arch: x64,    storage_size_gib: 300, boot_image: github-ubuntu-2004,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-arm,                         vm_size: standard-2,  arch: arm64,  storage_size_gib: 86,  boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-2-arm,              vm_size: standard-2,  arch: arm64,  storage_size_gib: 86,  boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-4-arm,              vm_size: standard-4,  arch: arm64,  storage_size_gib: 150, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-8-arm,              vm_size: standard-8,  arch: arm64,  storage_size_gib: 200, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-16-arm,             vm_size: standard-16, arch: arm64,  storage_size_gib: 300, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-2-arm-ubuntu-2204,  vm_size: standard-2,  arch: arm64,  storage_size_gib: 86,  boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-4-arm-ubuntu-2204,  vm_size: standard-4,  arch: arm64,  storage_size_gib: 150, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-8-arm-ubuntu-2204,  vm_size: standard-8,  arch: arm64,  storage_size_gib: 200, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-16-arm-ubuntu-2204, vm_size: standard-16, arch: arm64,  storage_size_gib: 300, boot_image: github-ubuntu-2204,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-2-arm-ubuntu-2004,  vm_size: standard-2,  arch: arm64,  storage_size_gib: 86,  boot_image: github-ubuntu-2004,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-4-arm-ubuntu-2004,  vm_size: standard-4,  arch: arm64,  storage_size_gib: 150, boot_image: github-ubuntu-2004,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-8-arm-ubuntu-2004,  vm_size: standard-8,  arch: arm64,  storage_size_gib: 200, boot_image: github-ubuntu-2004,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-16-arm-ubuntu-2004, vm_size: standard-16, arch: arm64,  storage_size_gib: 300, boot_image: github-ubuntu-2004,     location: github-runners, enable_gpu: false }
+- { name: ubicloud-standard-6-gpu,              vm_size: standard-6,  arch: x64,    storage_size_gib: 180, boot_image: github-gpu-ubuntu-2204, location: github-runners, enable_gpu: true  }
```

Comment:
Does the fact that these has gpu make this non-standard configuration? i.e. should the name be ubicloud-gpu-6?

Or do you think we would have standard-gpus and non-standard-gpus in the future?

Reply:
We will probably have non-standard-gpus for higher spec usecases. This comment made me think about the naming though. I changed the label to be `ubicloud-standard-gpu-6`